awstest/scenarios/apigateway.ftl

```diff
@@ -0,0 +1,127 @@
+                "Settings" : {
+                    "Registries": {
+                        "openapi": {
+                            "EndPoint": "account-registry-abc123",
+                            "Prefix": "openapi/"
+                        }
+                    }
+                }
+            },
+            {
+                "Type" : "Builds",
+                "Scope" : "Products",
+                "Namespace" : "mockedup-integration-aws-apigateway-base",
+                "Settings" : {
+                    "COMMIT" : "123456789#MockCommit#",
+                    "FORMATS" : ["openapi"]
+                }
+            },
+            {
+                "Type" : "Settings",
+                "Scope" : "Products",
+                "Namespace" : "mockedup-integration-aws-apigateway-base",
+                "Settings" : {
+                    "apigw": {
+                        "Internal": true,
+                        "Value": {
+                            "Type": "lambda",
+                            "Proxy": false,
+                            "BinaryTypes": ["*/*"],
+                            "ContentHandling": "CONVERT_TO_TEXT",
+                            "Variable": "LAMBDA_API_LAMBDA"
+                        }
+                    }
+                }
+            }
+        ]
+        blueprint={
+            "Tiers" : {
+                "app" : {
+                    "Components" : {
+                        "apigatewaybase" : {
+                            "apigateway" : {
+                                "Instances" : {
+                                    "default" : {
+                                        "DeploymentUnits" : ["aws-apigateway-base"]
+                                    }
+                                },
+                                "IPAddressGroups" : [ "_global" ],
+                                "Profiles" : {
+                                    "Testing" : [ "apigatewaybase" ]
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "TestProfiles" : {
+                "apigatewaybase" : {
+                    "apigateway" : {
+                        "TestCases" : [ "apigatewaybase" ]
+                    }
+                }
+            },
+            "TestCases" : {
+                "apigatewaybase" : {
+                    "OutputSuffix" : "template.json",
+                    "Structural" : {
+                        "CFN" : {
+                            "Resource" : {
+                                "RestApi" : {
+                                    "Name" : "apiXappXapigatewaybase",
+                                    "Type" : "AWS::ApiGateway::RestApi"
+                                },
+                                "Deployment" : {
+                                    "Name" : "apiDeployXappXapigatewaybaseXj2xuero56t",
```

Comment:
Are we fixing the runid? The deployment name id changes each time so feels like the test will fail...

Reply:
Good point, I've locked the runId in the https://github.com/hamlet-io/engine/pull/1373 and updated this PR to align with that fixed run Id